addons/pandora/model/property.gd

```diff
@@ -134,6 +134,9 @@ func save_data() -> Dictionary:
 		data["_setting_overrides"] = _setting_overrides
 	return data
 
+func equals(other: PandoraProperty) -> bool:
```

Comment:
This won't work as we create wrapper property objects to orchestrate the overriding logic. Instead of doing e.g.:
```gdscript
_name == other._name
```
do this:
```gdscript
get_entity_name() == other.get_entity_name()
```
You may even be able to verify my claim by writing a test in entity backend that has overriding properties and you attempt to do equals checks on properties between different categories on the inheritance chain.